mapactionpy_arcmap/MapChef.py

```diff
@@ -56,25 +58,36 @@ def cook(self, productName, countryName):
             properties = self.layerProperties.get(layer)
             # Add layer to the report for later
             mapResult = MapResult(layer)
-            if (properties is not None):             
+            if (properties is not None):
                 layerFilePath = os.path.join(self.layerDirectory, (properties.layerName + ".lyr"))
                 if (os.path.exists(layerFilePath)):
                     self.dataFrame = arcpy.mapping.ListDataFrames(self.mxd, properties.mapFrame)[0]
                     layerToAdd = arcpy.mapping.Layer(layerFilePath)
-                    dataFilePath= os.path.join(self.root, "GIS", "2_Active_Data", properties.sourceFolder)
+                    # APS 06/09/2019 Please remove hardcoded path. See github comment
+                    dataFilePath = os.path.join(self.root, "GIS", "2_Active_Data", properties.sourceFolder)
                     if (os.path.isdir(dataFilePath)):
+                        # APS 06/09/2019 I do not understand what you are testing here
+                        # (eg ` if ("/" not in properties.regExp):`).
+                        # Why is a forward slash required in a regex for a filename/featureclass name?
                         if ("/" not in properties.regExp):
```

Comment:
I do not understand what you are testing here (eg ` if ("/" not in properties.regExp):`). Why is a forward slash required in a regex?